packages/one-app-dev-bundler/esbuild/utils/server-style-aggregator.js

```diff
@@ -14,14 +14,36 @@
  * permissions and limitations under the License.
  */
 
-let aggregatedStyles = '';
+let aggregatedStyles = {
+  deps: [],
+  local: [],
+};
 
-export function addStyle(style) {
-  aggregatedStyles += style;
+const sheetDigests = new Set();
+
+export function addStyle(digest, css, isDependencyFile) {
+  if (!sheetDigests.has(digest)) {
+    sheetDigests.add(digest);
+
+    aggregatedStyles[isDependencyFile ? 'deps' : 'local'].push({
+      css,
+      digest,
+    });
+  }
 }
 
-export const getAggregatedStyles = () => aggregatedStyles;
+/**
+ * Returns aggregated styles object from all parsed CSS files
+ * @returns {{deps: *[{css: string, digest: string}], local: *[{css: string, digest: string}]}}
```

Comment:
This is returning a string not object. 

```suggestion
 * @returns {string}
```

Reply:
[c57043b](https://github.com/americanexpress/one-app-cli/commit/c57043b6c5e50b41f3c7da5ff35ed1d0f41e099a)